app/scripts/templates/mainmenu.eco

```diff
@@ -105,6 +105,7 @@
         <ul>
             <li><a >Help Page</a></li>
             <li><a >Dative Documentation</a></li>
+            <li><a >Version <%= @version %></a></li>
         </ul>
```

Comment:
did the PI's request a desktop feel to the app? it might mean we have to plan to make a separate app for mobile since most desktop css emulation libraries arent very good at small screen sizes (or touch events) we do have a a substantial number of users who use tablets to access the app while in class...